playbook/app/pb_kits/playbook/pb_text_input/_text_input.tsx

```diff
@@ -70,12 +70,15 @@ const TextInput = (props: TextInputProps, ref: React.LegacyRef<HTMLInputElement>
   const shouldShowAddOn = icon !== null
   const addOnCss = shouldShowAddOn ? 'text_input_wrapper_add_on' : ""
   const addOnDarkModeCardCss = (shouldShowAddOn && dark) ? 'add-on-card-dark' : ""
+  const defaultMarginBottom = props.marginBottom ?? "mb_sm"
+
   const css = classnames([
     'pb_text_input_kit',
     inline ? 'inline' : "",
     error ? 'error' : "",
     globalProps(props),
     className,
+    defaultMarginBottom
```

Comment:
```suggestion
    defaultMarginBottom,
    className
```

`className` must always come last or it will break places in nitro-web where we use CSS Modules.